system/Cache/CacheInterface.php

```diff
@@ -34,6 +36,19 @@ public function get(string $key);
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * Get an item from the cache, or execute the given Closure and store the result.
+	 *
+	 * @param string  $key      Cache item name
+	 * @param integer $ttl      Time to live
+	 * @param Closure $callback Callback return value
+	 *
+	 * @return mixed
+	 */
+	public function remember(string $key, int $ttl, Closure $callback);
+
+	//--------------------------------------------------------------------
+
```

Comment:
Adding a new function to the `CacheInterface` is a breaking change. I suggest extending this interface, say `RememberableCacheInterface`, with the new `remember` method and the existing handlers will extend this instead.

Reply:
Thank you @paulbalandan, i will change this asap.